ts/parts/Chart.ts

```diff
@@ -33,6 +33,9 @@ declare global {
             plotBox: BBoxObject;
             spacingBox: BBoxObject;
         }
+        interface CaptionObject extends SVGElement {
```

Comment:
Is this used anywhere, otherwise it should be removed?

Reply:
It is equivalent to `SubtitleObject`. I forgot to add doclets for `CaptionObject` and `CaptionObject#update`.